tests/test_enterprise_learner_portal/api/test_views.py

```diff
@@ -134,3 +133,121 @@ def test_view_requires_openedx_installation(self):
                     enterprise_id=str(self.enterprise_customer.uuid)
                 )
             )
+
+        with self.assertRaises(NotConnectedToOpenEdX):
+            query_params = {
+                'enterprise_id': str(self.enterprise_customer.uuid),
+                'course_id': self.course_run_id,
+                'marked_done': True,
+            }
+
+            self.client.patch(
+                '{host}{path}?{query_params}'.format(
+                    host=settings.TEST_SERVER,
+                    path=reverse('enterprise-learner-portal-course-enrollment-list'),
+                    query_params=urlencode(query_params),
+                )
+            )
+
+    @mock.patch('enterprise_learner_portal.api.v1.views.EnterpriseCourseEnrollmentSerializer')
+    @mock.patch('enterprise_learner_portal.api.v1.views.get_course_overviews')
+    def test_patch_success(self, mock_get_overviews, mock_serializer):
+        """
+        View should update the enrollment's marked_done field and return serialized data from
+         EnterpriseCourseEnrollmentSerializer for the enrollment (which we mock in this case)
+        """
+        mock_get_overviews.return_value = {'overview_info': 'this would be a larger dict'}
+        mock_serializer.return_value = self.MockSerializer()
+        query_params = {
+            'enterprise_id': str(self.enterprise_customer.uuid),
+            'course_id': self.course_run_id,
+            'marked_done': True,
+        }
```

Comment:
Just a suggestion, but as a sanity check, maybe assert that `marked_done` is False before you hit the patch endpoint